rrule-wasm/tests/wasm.rs

```diff
@@ -0,0 +1,11 @@
+use wasm_bindgen_test::*;
+use rrule_wasm::get_all_date_recurrences;
+
+#[wasm_bindgen_test]
+fn test_date_recurrences() {
```

Comment:
Not sure what value this test brings over the existing test suite we have.

Reply:
At this moment I agree with you. Do you prefer I remove it?

Reply:
yeah, can be removed, thanks